### PR TITLE
Daniil/Mt5 missing download option

### PR DESCRIPTION
**Pull request by Daniil-Deriv**

### CODE REVIEW

File: src/templates/static/metatrader/download.jsx [hunk 1]
```diff
@@ -104,13 +104,15 @@ const DesktopDownloadBlock = ({
 );
 
 const MT5DesktopApp = ({ is_first_child, has_desktop_app }) => {
-    const txt_unsupported = it.L('The MT5 desktop app is not supported by macOS, Windows XP, Windows 2003, and Windows Vista.');
+    const txt_unsupported = it.L('The MT5 desktop app is not supported by Windows XP, Windows 2003, and Windows Vista.');
```
Comment (mustofa-binary) [inline src/templates/static/metatrader/download.jsx:107]: we do support macOs, any reason as of why do we remove it?

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/javascript/app/pages/resources/metatrader/download.ui.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ const toggleDownloadPage = target => {
         getElementById(`${target}-description`).setVisibility(1);
     }
     getElementById(`mt5_download_${target === 'mac' ? 'mac_' : ''}platforms`).setVisibility(1);
+    getElementById(`mt5_download_${target !== 'mac' ? 'mac_' : ''}platforms`).setVisibility(0);
```
Comment (Bahar-FS) [inline src/javascript/app/pages/resources/metatrader/download.ui.js:29]: Don't both of these lines mean the same?

File: src/templates/static/metatrader/download.jsx (hunk 1 above)
Comment (Daniil-Deriv) [reply src/templates/static/metatrader/download.jsx:107]: @mustofa-binary don't know why it was ommited, but as I understand, download option for mac should be added to binary now

**Review by Bahar-FS — CHANGES_REQUESTED** [review]

File: src/javascript/app/pages/resources/metatrader/download.ui.js (hunk 2 above)
Comment (Daniil-Deriv) [reply src/javascript/app/pages/resources/metatrader/download.ui.js:29]: No, second string resolves the component duplication problem. 
image.   What happens without it

File: src/templates/static/metatrader/download.jsx (hunk 1 above)
Comment (Bahar-FS) [reply src/templates/static/metatrader/download.jsx:107]: @mustofa-binary It seems that some months ago showing the download for mac removed, and now the QA asked for that.
--https://github.com/binary-com/binary-static/pull/5965/files

**Review by Bahar-FS — APPROVED** [review]

**Pull request merged**